app/models/keyword.rb

```diff
@@ -12,6 +12,7 @@ def to_s
   end
 
   before_save do
+    self.term = self.term.gsub(Madek::Constants::TRIM_WHITESPACE_REGEXP, '')
```

Comment:
`self` is necessary only for assigning a value, getting a value with `self` is redundant ;)